unity/CAVE/Assets/W3D_Translator/Classes.cs

```diff
@@ -182,6 +184,24 @@ public enum ContentTypes {
             [XmlEnum("Model")] Model,
             [XmlEnum("Light")] Light,
             [XmlEnum("ParticleSystem")] ParticleSystem,
+        }
+
+        public GameObject Create(Object parent) {
+            GameObject gameObject = this.ContentData switch
+            {
+                Text text => text.GenerateTMP(
+                    parent.LinkRoot is not null, 
+                    Xml.ConvertColor(parent.ColorString)
+                ),
+                Image image => new GameObject(), // TODO (65)
+                StereoImage stereoImage => new(), // TODO (66)
+                Model model => new GameObject(), // TODO (67)
+                Light light => new GameObject(), // TODO (68)
+                ParticleSystem particleSystem => new GameObject(), // TODO (69)
+                _ => new GameObject(), // TODO: - Shouldn't occur, throw error
+            };
```

Comment:
Look at that pattern matching! Looks good!

Reply:
So much easier!